shared/src/hooks/useENS.ts

```diff
@@ -0,0 +1,23 @@
+import { useWeb3React } from "@web3-react/core";
+import { ethers } from "ethers";
+import { useEffect, useState } from "react";
+import { CHAINID } from "../utils/env";
+
+const useENS = (address: string) => {
```

Comment:
I don't see `useENS` used anywhere btw. We also already have `useENSSearch`

Reply:
I’m not what what you’re suggesting. There’s no ENS forks on Aurora.

Reply:
@kenchangh  @fqlx  I think we can delete `useENS`. I have deleted the file in this [commit](https://github.com/ribbon-finance/ribbon-frontend/commit/10fccf2f5fc8db3cad8585afc95e098c94ca6483), not sure how does it end up here again

Reply:
@cjinghong Thanks, I must've accidentally `git add` this file during a rebase. Deleted.